### PR TITLE
Remove alculquicondor and Huang-Wei from OWNERS

### PR DESCRIPTION
**Pull request by alculquicondor**
#### What type of PR is this?

/kind documentation

#### What this PR does / why we need it:

I never had any significant involvement in this repo and expect to have less chances of involvement in the future.

#### Which issue(s) this PR fixes:

Fixes #

#### Special notes for your reviewer:

/label tide/merge-method-squash

### CODE REVIEW

File: OWNERS [hunk 1]
```diff
@@ -1,7 +1,6 @@
 # See the OWNERS docs at https://go.k8s.io/owners
 
 approvers:
-- alculquicondor
 - Huang-Wei
```
Comment (Huang-Wei) [inline OWNERS:4]: @alculquicondor could you help also remove me from both approvers and reviewers? 🙏

Comment (alculquicondor) [reply OWNERS]: Done

**Review by sanposhiho — APPROVED** [review]
/lgtm
/approve

**Comment by sanposhiho** [comment]
/retitle  Remove alculquicondor and Huang-Wei from OWNERS

**Pull request merged**